src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -89,63 +89,63 @@ def __init__(
     def initialize_datamodel(
         self, request: DataModelProtoModule.InitDatamodelRequest
     ) -> DataModelProtoModule.InitDatamodelResponse:
-        """initDatamodel rpc of DataModel service."""
+        """InitDatamodel rpc of DataModel service."""
         return self._stub.initDatamodel(request, metadata=self._metadata)
 
     @catch_grpc_error
     def get_attribute_value(
         self, request: DataModelProtoModule.GetAttributeValueRequest
     ) -> DataModelProtoModule.GetAttributeValueResponse:
-        """getAttributeValue rpc of DataModel service."""
+        """GetAttributeValue rpc of DataModel service."""
```

Comment:
```suggestion
        """GetAttributeValue RPC of DataModel service."""
```
RPC also should be capitalized

Are there any other acronyms/abbreviations that we may be missing?

There are a few places in docstrings with "gprc", should be capitalized as gRPC: https://grpc.io/